analysis2_manuscript.tex

```diff
@@ -334,7 +334,10 @@ \section*{Implementing Selection in \stdpopsim}
     %IG: I moved this label to the newly added methods section --> \revpoint{1}{5}
     Each \texttt{Annotation} object represents a collection of functional
     genomic elements (e.g., exons, introns, intergenic regions, etc.)
-    whose coordinates are specified using a GFF3 annotation file. \revpoint{2}{1}
+    whose coordinates are specified using a GFF3 annotation file. 
+    For the purposes of \stdpopsim, we do not allow for overlapping annotations -- 
+    genomic elements of the given annotation being modeled (e.g., CDS) are prioritized over overlapping annotations of other types (e.g., exons)
```

Comment:
I don't understand what this line means?

Reply:
Ah - I think this is what you are trying to say?
<img width="1080" height="287" alt="Screenshot From 2025-07-29 15-34-53" src="https://github.com/user-attachments/assets/88ece70c-9ea9-4683-8d6f-d5400d79a7bc" />